doc/source/cluster/kubernetes/user-guides/kubectl-plugin.md

```diff
@@ -0,0 +1,65 @@
+(kubectl-plugin)=
+
+# Use Kubectl Plugin (alpha)
+
+Starting from KubeRay v1.2.2, you can use the `kubectl ray` plugin to simplify common workflows when deploying Ray on Kubernetes. If you aren't familiar with Kubernetes, this plugin simplifies running Ray on Kubernetes.
+
+## Installation
+
+See [KubeRay Kubectl Plugin](https://github.com/ray-project/kuberay/tree/master/kubectl-plugin) to install the plugin.
+
+Install the Kuberay kubectl plugin using one of the following methods:
+
```

Comment:
```suggestion

- Install using Krew `kubectl` (recommended)
- Download from GitHub releases

```

Reply:
I meant to add this comment. If you can include this, that would be nice. Thanks!